experimental/postgrest/Postgrest_Dockerfile

```diff
@@ -25,7 +25,7 @@
 FROM ubuntu:15.10
 MAINTAINER dev@trafficcontrol.apache.org
 
-RUN apt-get install -y curl tar xz-utils postgresql-client
+RUN apt-get --no-install-recommends install -y curl tar xz-utils postgresql-client
```

Comment:
I would prefer disabling `APT::Install-Recommends` and `APT::Install-Suggests` in the APT configuration to passing `--no-install-recommends` for every apt or apt-get command.